ush/config_defaults.yaml

```diff
@@ -2811,63 +2107,27 @@ verification:
   OBS_NDAS_SFCorUPA_FN_TEMPLATE: 'prepbufr.ndas.{valid?fmt=%Y%m%d%H}'
   OBS_NDAS_SFCorUPA_FN_METPROC_TEMPLATE: '${OBS_NDAS_SFCorUPA_FN_TEMPLATE}.nc'
   #
-  # VX_LOCAL_MODULE_FN:
-  # Name (without extension) of the local module file for running the vx
-  # tasks in the workflow.
-  #
-  VX_LOCAL_MODULE_FN: 'run_vx'
-  #
-  # RUN_TASKS_METVX_DET:
-  # Flag that specifies whether to run deterministic verification.  If set
-  # to True, this will run deterministic vx on the post-processed forecast
-  # output.  This post-processed output may consist of a single forecast
-  # or an ensemble of foreasts, and it may be staged from previous runs of
-  # the SRW App or may be generated by running the TN_RUN_FCST task as part
-  # of the current SRW-App-generated experiment.
-  #
-  # RUN_TASKS_METVX_ENS:
-  # Flag that specifies whether to run ensemble verification.  The ensemble
-  # forecast output on which vx will be run may be staged or generated by
-  # running an ensemble of forecasts with the weather model as part of the
-  # current SRW-App-generated experiment.
-  #
```

Comment:
@mkavulich Since these switches are removed, how do we now turn deterministic and ensemble vx on/off?

Reply:
@mkavulich Looks like these are now determined by the `taskgroups` entry in the configuration file, right?